modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -423,9 +423,9 @@ label:deprecated[Deprecated in 5.15]
 |===
 
 [[discovery-metrics]]
-=== Discovery metrics
+=== Discovery metrics v1
 
-.Discovery metrics
+.Discovery metrics v1
```

Comment:
If you want to keep it, you need to change it in the code as well.

Reply:
Done here -> https://github.com/neo-technology/neo4j/pull/26065